scanner/updater/export.go

```diff
@@ -62,30 +67,44 @@ func Export(ctx context.Context, outputDir string) error {
 	outOfTree = append(outOfTree, updaterSet.Updaters())
 
 	for i, uSet := range [][]string{
-		{"oracle", "photon", "suse", "aws", "rhcc"},
-		{"alpine", "rhel", "ubuntu", "osv", "debian"},
+		{"oracle"}, {"photon"},
+		{"suse"}, {"aws"},
+		{"alpine"}, {"rhel"},
+		{"debian"}, {"rhcc"},
+		{"ubuntu"}, {"osv"},
 	} {
 		jsonStore, err := jsonblob.New()
 		if err != nil {
 			return err
 		}
+		var mgr *updates.Manager
+		if i >= len(outOfTree) {
```

Comment:
What happens when/if `len(outOfTree) > number of loop iterations`?

Reply:
updaterSet always has size of 1:
```
// UpdaterSet creates a new updater set with the provided vulnerability data.
func UpdaterSet(_ context.Context, vulns []*claircore.Vulnerability) (driver.UpdaterSet, error) {
	res := driver.NewUpdaterSet()
	err := res.Add(&updater{data: vulns})
	if err != nil {
		return res, err
	}
	return res, nil
}
```

Reply:
what happens if one day `len(outOfTree) == 20`? Then we will miss it. I think for now we should just add the out-of-tree in the first iteration, only. It accomplishes the same as now and ensures we don't miss any out-of-tree updaters in the future. Sure it doesn't spread the load if future updaters are added, but at least we know they are all utilized and we don't accidentally miss any in the future